dataactbroker/handlers/fileHandler.py

```diff
@@ -1046,12 +1046,20 @@ def serialize_submission(submission):
     else:
         submission_user_name = sess.query(User).filter_by(user_id=submission.user_id).one().name
 
+
+    cgac = sess.query(CGAC).\
+        filter_by(cgac_code=submission.cgac_code).one_or_none()
+    if cgac:
+        agency_name = cgac.agency_name
+    else:
+        agency_name = ''
+
     return {
         "submission_id": submission.submission_id,
         "last_modified": submission.updated_at.strftime('%Y-%m-%d'),
         "size": total_size,
         "status": status,
-        "errors": submission.number_of_errors,
```

Comment:
Talked with Kaitlin and added screenshots of the table with agency added. The table was crowded and hard to read so we discussed it and she suggested removing the number of errors. This gave the rest of the table space to become readable.
![screen shot 2017-02-01 at 11 55 37 am](https://cloud.githubusercontent.com/assets/4334164/22552555/6bc9e758-e927-11e6-99cd-ea7bc5bb4468.png)
![screen shot 2017-02-01 at 2 09 08 pm](https://cloud.githubusercontent.com/assets/4334164/22552557/6dbecc22-e927-11e6-84e4-5381e735f12e.png)